### PR TITLE
Add @willium as maintainer

### PR DESCRIPTION
**Pull request by domoritz**

### CODE REVIEW

**Review by mattijn — APPROVED** [review]

**Comment by jonmmease** [comment]
Is this blocked on anything @domoritz?

**Comment by domoritz** [comment]
Ups, no.

**Comment by domoritz** [comment]
Yeah, @willium should approve officially.

**Review by willium — APPROVED** [review]
Excited to be on the team :)

**Pull request merged**
